src/Pages/SignUpPage/SignUpPage.tsx

```diff
@@ -8,9 +8,22 @@ const SignUpPage: React.FC = () => {
     const [username, setUsername] = useState('');
     const [password, setPassword] = useState('');
     const [isCheckboxChecked, setIsCheckboxChecked] = useState(false);
-
+    const [passwordError, setPasswordError] = useState('');
+    const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const { value } = e.target;
+        setPassword(value);
+        if (value.length < 8) {
+            setPasswordError('Password must be at least 8 characters long');
+        } else {
+            setPasswordError('');
```

Comment:
@rooshnie23 you could set the password within this else statement or outside this if-else block altogether